cmd/containerd-shim-runhcs-v1/task_test.go

```diff
@@ -91,6 +92,19 @@ func (tst *testShimTask) DumpGuestStacks(ctx context.Context) string {
 	return ""
 }
 
+func (tst *testShimTask) Update(ctx context.Context, req *task.UpdateTaskRequest) error {
+	data, err := typeurl.UnmarshalAny(req.Resources)
+	if err != nil {
+		return errors.Wrapf(err, "failed to unmarshal resources for container %s update request", req.ID)
+	}
+	if _, ok := data.(*specs.WindowsResources); ok {
+		return nil
+	} else if _, ok := data.(*specs.LinuxResources); ok {
+		return nil
+	}
+	return errors.New("update resources must be of type *WindowsResources or *LinuxResources")
+}
+
```

Comment:
Can't we just call this? https://github.com/microsoft/hcsshim/pull/931/files#diff-53a84325b543dc2470551cd63a0e19f15fd4a02f20c8c5c946f69ec1fa23c54bR105